### ISSUE TITLE
{Freeway-Bug}[AdaptiveCards- Product Video]: Captions or transcription is not available while playing the pre-recorded video.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8366]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/samples/ProductVideo.html
OS Version: 22H2(OS Build 25300.1000)
Browser Version: Edge Dev Version 111.0.1661.36 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**

1. Open the above URL in the browser.
2. Navigate to 'Product Video' and press enter key to activate it.
3. Navigate to product video sample and press enter key to activate it.
4. Navigate the video control and observe the issue.

**Actual Result:**
Captions or transcription options are not present in the cards under the product video.

**Expected Result:**
Captions or transcription options should be available for the video under product video.

**User Experience:**
A person who can't listen to the voice after playing the video will find difficulty understanding voice if 'sub title' and Transcript' will not be available.

**WCAG Reference:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EVcPxC3kj3ZPhu1c2kGo9PoBEo0SF5vzBMaPBgVs2Sq6tw?e=zR44ln

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

![MAS7 3-Captions or transcription is not available while playing the pre-recorded video](https://user-images.githubusercontent.com/95913748/224992566-12de4434-3ce6-4e00-b2de-6b27d7dcbfd8.png)

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @vagpt,

Closing this issue because it does not repro on the latest release of our website. Let me know if you have any questions!

**Comment by vagpt** [comment]
This issue is working fine on the environment below.

**Test Environment:**
URL: https://adaptivecards.io/samples/ProductVideo.html
Browser Version: 114.0.1807.1
OS Version: 22H2 (OS Build 25330.1000)
